src/languages/gherkin.js

```diff
@@ -9,10 +9,24 @@ export default function(hljs) {
   return {
     name: 'Gherkin',
     aliases: [ 'feature' ],
-    keywords: 'Feature Background Ability Business\ Need Scenario Scenarios Scenario\ Outline Scenario\ Template Examples Given And Then But When',
+    keywords: {
+      // Custom pattern to support keywords with spaces and those ending with a colon
+      $pattern: /[A-Z][a-z]+(?: [A-Z][a-z]+)?:?/,
```

Comment:
This is going to break and have weird edge cases like `And When`... which would match BOTH words, and then not highlight any keywords because all we do is scan the text with this regex looking for POTENTIAL keywords.

Multi-word keywords are better handled with modes I think.